src/applications/find-forms/tests/helpers/index.unit.spec.js

```diff
@@ -113,4 +117,14 @@ describe('Find VA Forms helpers', () => {
       sortedResultsNodesTextByLatestRevisionOldest,
     );
   });
+
+  it('regexpDashAdder adds the dash where it should', () => {
+    const transformString = regexpDashAdder('20213', 2);
+    expect(transformString).to.eql('20-213');
+  });
+
+  it('transformSearchTerm adjusts the term to allow for faulty searches', () => {
+    const transformString = transformSearchTerm('1010');
+    expect(transformString).to.eql('10-10');
+  });
```

Comment:
The way this function works with a direct input and a direct output makes it golden for a strong unit test. You could list a whole bunch of inputs/output in an array and iterate across them all.

```js
const searchQueryAutoCorrectResults = new Map([
    ['1010', '10-10'],
    // ... tons of inputs/outputs
]);

for (const [term, correctedTerm] of searchQueryAutoCorrectResults) {
    it('produces the expected auto-corrected result', () => {
        const result = transformSearchTerm(term);
        expect(result).to.equal(correctedTerm);
    )
}
```

Great opportunity to have your unit test reflect exactly all of the different input/output edge cases you may have tested with manually

Reply:
Oh wow, that's a great point, we can take it a step further!

Reply:
@ncksllvn This is ready for another review when you get the chance.

Reply:
Well done!!